generated/datafactory/microsoft.datafactory/2018-06-01/types.md

```diff
@@ -5678,13 +5867,16 @@ request-header-name-n:request-header-value-n Type: string (or Expression with re
 * **annotations**: any[]: List of tags that can be used for describing the trigger.
 * **description**: string: Trigger description.
 * **runtimeState**: 'Disabled' | 'Started' | 'Stopped' | string (ReadOnly): Indicates if trigger is running or not. Updated when Start/Stop APIs are called on the Trigger.
+
 ### BlobEventsTrigger
 #### Properties
+* **pipelines**: [TriggerPipelineReference](#triggerpipelinereference)[]: Pipelines that need to be started.
```

Comment:
Indirect inheritance: [`pipelines` is defined on `MultiplePipelineTrigger`](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/Trigger.json#L67), which is the direct parent of the enum branch and direct child of the enum base.